src/connection/session-manager.js

```diff
@@ -2,13 +2,23 @@ import {
   STORAGE_SESSION_NAME
 } from '../constants'
 
-class Session {
+class SessionManager {
+
+  constructor(type) {
+    this.type = type
+  }
 
   _getStorage() {
-    if(window.localStorage) {
-      return window.localStorage
+    // Chrome can return Access Denied error when trying to access window.localStorage
```

Comment:
What can customers do about it? Isn't in for private mode only?

Reply:
yeah, forgot to mention private mode